backend/src/test/java/com/github/esgoet/backend/services/BookServiceUnitTest.java

```diff
@@ -69,7 +72,21 @@ void getBook_Test_whenBookDoesNotExists_thenThrow() {
         verify(bookRepo).findById("1");
     }
 
-
+    @Test
+    void addABookTest_whenNewBookAsInput_thenReturnNewBook() {
+        // GIVEN
+        NewBookDto newBookDto = new NewBookDto("J. K. Rowling", "Harry Potter", Genre.FANTASY, localDate);
+        Book bookToSave = new Book(idService.randomId(), newBookDto.author(), newBookDto.title(), newBookDto.genre(), newBookDto.publicationDate());
+        when(bookRepo.save(bookToSave)).thenReturn(bookToSave);
+
+        // WHEN
+        Book actual = bookService.saveNewABook(newBookDto);
+
+        // THEN
+        Book expected = bookToSave;
```

Comment:
i think it's better practice to create a new book for expected, rather than use the other book, to make sure the .isEqual() method is working correctly
```suggestion
        Book expected = new Book(new Book(idService.randomId(), "J. K. Rowling", "Harry Potter", Genre.FANTASY, localDate);
```